### DISCUSSION TITLE
posts recents

### DISCUSSION THREAD

**Discussion by douglastos** [DaniloCarSan/danilocarsan.github.io#12] [General]
dei uma olhada nos seus commits... e já visualizei seu endereço e aqui não consigo reproduzir os "posts recents" o que será que eu estou errando?

**Comment by DaniloCarSan** [comment]
Opa beleza, então você precisa ir no seu arquivo de configuração de layout do quartz e adicionar o componente.

Linha: 36
https://github.com/DaniloCarSan/danilocarsan.github.io/blob/main/quartz.layout.ts


![image](https://github.com/DaniloCarSan/danilocarsan.github.io/assets/29821188/0702b5dd-b0fb-4ead-b228-5d7a1c233de9)


No seu:

https://github.com/douglastos/douglastos.github.io/blob/main/quartz.layout.ts

**Comment by douglastos** [comment]
Boa man obrigado por ser solicito novamente funcionou sim!
